frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -208,21 +200,25 @@ const AdminShoutouts: React.FC = () => {
           <ChooseDate dateField={earlyDate} dateFunction={setEarlyDate} />
           <ChooseDate dateField={lastDate} dateFunction={setLastDate} />
           <Button.Group className={styles.buttonGroup}>
-            <ButtonPiece shoutoutList={allShoutouts} buttonText={'ALL'} />
-            <ButtonPiece
-              shoutoutList={allShoutouts.filter((shoutout) => shoutout.hidden)}
-              buttonText={'HIDDEN'}
-            />
-            <ButtonPiece
-              shoutoutList={allShoutouts.filter((shoutout) => !shoutout.hidden)}
-              buttonText={'PRESENT'}
-            />
+            <Button color={view === 'ALL' ? 'blue' : 'grey'} onClick={() => setView('ALL')}>
+              ALL
+            </Button>
+            <Button color={view === 'HIDDEN' ? 'blue' : 'grey'} onClick={() => setView('HIDDEN')}>
+              HIDDEN
+            </Button>
+            <Button color={view === 'PRESENT' ? 'blue' : 'grey'} onClick={() => setView('PRESENT')}>
+              PRESENT
+            </Button>
           </Button.Group>
         </Form.Group>
       </Form>
       <div className={styles.shoutoutsListContainer}>
-        <ListTitle />
-        <DisplayList />
+        <Header className={styles.formTitle} content={getListTitle(view)}></Header>;
```

Comment:
there is a floating semicolon under each title, "all shoutouts", "hidden shoutouts" etc, in the admin shoutouts, i think that semicolon after </Header> may be the cause